File: test/corpus/basic/01-simple-color/case.ts
import { newOutput, Profile, rgbColor, withProfile } from '../../../../src/index.js';

// Force TrueColor profile for consistent testing
const out = newOutput(process.stdout, withProfile(Profile.TrueColor));

// Test basic color rendering
const red = out.string('Red Text').foreground(rgbColor('#FF0000'));
const blue = out.string('Blue Text').foreground(rgbColor('#0000FF'));
const green = out.string('Green Text').foreground(rgbColor('#00FF00'));

console.log(red.toString());
console.log(blue.toString());
console.log(green.toString());

File: test/corpus/basic/01-simple-color/case.go
package main

import (
	"fmt"

	"github.com/muesli/termenv"
)

func main() {
	// Force TrueColor profile for consistent testing
	out := termenv.NewOutput(nil, termenv.WithProfile(termenv.TrueColor))
	
	// Test basic color rendering
	red := out.String("Red Text").Foreground(out.Color("#FF0000"))
	blue := out.String("Blue Text").Foreground(out.Color("#0000FF"))
	green := out.String("Green Text").Foreground(out.Color("#00FF00"))
	
	fmt.Println(red)
	fmt.Println(blue)  
	fmt.Println(green)
}